src/Raven.Client/Documents/Operations/Backups/Sharding/ShardRestoreSettings.cs

```diff
@@ -38,7 +38,7 @@ public class SingleShardRestoreSetting : IDynamicJson
     {
         public int ShardNumber { get; set; }
         public string NodeTag { get; set; }
-        public string BackupPath { get; set; }
+        public string FolderName { get; set; }
```

Comment:
this cannot point to a single file, correct?

Reply:
actually I think it will also work if it points to a single file.. at least on google cloud it does (didn't check on s3 or azure)
same thing in regular restore, we can have `GoogleCloudSettings.RemoteFolderName` pointing to a single file and it will work

you think I should revert back to `BackupPath` ?

Reply:
I'm merging this PR, @aviv86 I would like to have consistent naming, what it should be?